lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -183,19 +183,22 @@ public function listTableIndexes($table)
     /**
      * Returns true if all the given tables exist.
      *
-     * @param string[] $tableNames
+     * @param string|string[] $tableNames
+     * @param bool            $filtered
      *
      * @return bool
      */
-    public function tablesExist($tableNames)
+    public function tablesExist($tableNames, $filtered = true)
```

Comment:
This looks like a BC-break for extending classes that override this method. I think you should use `func_get_args()` to retrieve that argument when passed.

Reply:
I covered it with tests, the original operation is preserved. I think it's unnecessary to call `func_get_args`. It only increases complexity.

Reply:
Doesn't make it less of a BC-break. Let me illustrate my point: https://3v4l.org/g4Er6
:boom: warning!